### ISSUE TITLE
BUG | Button Gradient Missing on Site Settings

### ISSUE THREAD

**Issue by maurocarreraok** [elementor/elementor#14731]
## Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

## Description

The button gradient is missing on the Site Settings configuration.

![image](https://user-images.githubusercontent.com/68023943/116238030-448c0900-a737-11eb-87d4-0e70efe20c8c.png)

## Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a default WordPress theme active.
- [X] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Source distribution v5.7.32-35
	PHP Version: 7.3.27
	PHP Max Input Vars: 3000
	PHP Max Post Size: 5120M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.7.1
	Site URL: https://staging4.*confidential*.com
	Home URL: https://staging4.*confidential*.com
	WP Multisite: No
	Max Upload Size: 5 GB
	Memory limit: 768M
	Permalink Structure: /%category%/%postname%/
	Language: en-US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor Child
	Version: 1.0.1
	Author: Elementor Team
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.3.1
	Parent Theme Author: Elementor Team

== User ==
	Role: 
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.93 Safari/537.36

== Active Plugins ==
	All-in-One WP Migration
		Version: 7.42
		Author: ServMask

	Code Snippets
		Version: 2.14.1
		Author: Code Snippets Pro

	Display Category Post Count
		Version: 1.1
		Author: Amit Maskare

	Elementor
		Version: 3.2.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.2.1
		Author: Elementor.com

	Instructor Role
		Version: 3.5.4
		Author: WisdmLabs

	JetBlocks For Elementor
		Version: 1.2.8
		Author: Crocoblock

	JetBlog For Elementor
		Version: 2.2.13
		Author: Crocoblock

	JetElements For Elementor
		Version: 2.5.6
		Author: Crocoblock

	JetEngine
		Version: 2.7.8
		Author: Crocoblock

	JetSearch For Elementor
		Version: 2.1.10
		Author: Crocoblock

	JetSmartFilters
		Version: 2.2.2
		Author: Crocoblock

	JetTabs For Elementor
		Version: 2.1.12
		Author: Crocoblock

	JetTricks
		Version: 1.3.5
		Author: Crocoblock

	JSM's Show Post Metadata
		Version: 1.3.0
		Author: JS Morisset

	LearnDash LMS
		Version: 3.4.0.7
		Author: LearnDash

	LearnDash LMS - Certificate Builder
		Version: 1.0.0
		Author: LearnDash

	LearnDash LMS - Course Grid
		Version: 1.7.0
		Author: LearnDash

	LearnDash LMS - Elementor
		Version: 1.0.3
		Author: LearnDash

	LearnDash ProPanel
		Version: 2.1.4
		Author: LearnDash

	Make Column Clickable Elementor
		Version: 1.3.1
		Author: Fernando Acosta

	Plerdy
		Version: 1.04
		Author: Plerdy

	Quick Featured Images
		Version: 13.5.6
		Author: Kybernetik Services

	Rank Math SEO
		Version: 1.0.62
		Author: Rank Math

	Site Kit by Google
		Version: 1.31.0
		Author: Google

	User Registration
		Version: 1.9.6
		Author: WPEverest

	User Registration Advanced Fields
		Version: 1.4.1
		Author: WPEverest

	User Registration Content Restriction
		Version: 1.1.3
		Author: WPEverest

	User Registration Customize My Account
		Version: 1.1.1
		Author: WPEverest

	User Registration Extras
		Version: 1.0.5
		Author: WPEverest

	User Registration MailChimp
		Version: 1.2.2
		Author: WPEverest

	User Registration Profile Connect
		Version: 1.0.1
		Author: WPEverest

	User Registration Style Customizer
		Version: 1.0.3
		Author: WPEverest

	User Role Editor
		Version: 4.59
		Author: Vladimir Garagulya

	WP-Optimize - Clean, Compress, Cache
		Version: 3.1.8
		Author: David Anderson, Ruhani Rabin, Team Updraft

	WPvivid Backup Plugin
		Version: 0.9.53
		Author: WPvivid Team


== Must-Use Plugins ==
	Elementor Safe Mode
		Version: 1.0.0
		Author: Elementor.com

	LearnDash LMS Multisite
		Version: 1.0.0
		Author: LearnDash


== Elementor - Compatibility Tag ==
	
	Elementor Pro: header_not_exists
	JetBlocks For Elementor: header_not_exists
	JetBlog For Elementor: header_not_exists
	JetElements For Elementor: header_not_exists
	JetSearch For Elementor: header_not_exists
	JetTabs For Elementor: header_not_exists
	LearnDash LMS - Elementor: header_not_exists
	Make Column Clickable Elementor: header_not_exists

== Elementor Pro - Compatibility Tag ==
	


== Elements Usage ==
	
	archive : 2
		heading : 7
		image : 1
		jet-ajax-search : 2
		jet-blog-smart-tiles : 1
		jet-listing-dynamic-terms : 2
		jet-listing-grid : 4
		jet-smart-filters-checkboxes : 1
		jet-smart-filters-remove-filters : 1
		jet-smart-filters-search : 1
		jet-smart-filters-select : 1
		nav-menu : 1
		spacer : 2
	header : 1
		button : 5
		nav-menu : 2
		theme-site-logo : 2
	page : 1
		heading : 1
		image : 1
	popup : 10
		button : 1
		divider : 5
		heading : 2
		icon-box : 1
		icon-list : 2
		image : 2
		image-box : 2
		jet-button : 8
		share-buttons : 1
		video : 6
	product : 0
		woocommerce-breadcrumb : 1
		woocommerce-product-add-to-cart : 1
		woocommerce-product-data-tabs : 1
		woocommerce-product-images : 1
		woocommerce-product-meta : 1
		woocommerce-product-price : 1
		woocommerce-product-rating : 1
		woocommerce-product-related : 1
		woocommerce-product-short-description : 1
		woocommerce-product-title : 2
		woocommerce-product-upsell : 1
	footer : 1
		heading : 1
		image : 1
		nav-menu : 1
		social-icons : 1
	section : 30
		button : 5
		divider : 5
		heading : 26
		icon-box : 1
		icon-list : 24
		image : 27
		image-box : 2
		jet-button : 8
		spacer : 3
		template : 1
		text-editor : 1
	widget : 0
		button : 14
	single-post : 2
		heading : 4
		icon-list : 2
		image : 2
		jet-breadcrumbs : 2
		jet-listing-dynamic-link : 2
		jet-listing-dynamic-terms : 2
		jet-listing-grid : 2
		post-info : 2
		post-navigation : 2
		theme-post-content : 2
		theme-post-title : 2
	sfwd-topic : 1
		jet-slider : 1
		ld-course-content : 1
		ld-course-infobar : 1
		theme-post-content : 1
	sfwd-lessons : 2
		jet-slider : 1
		ld-course-content : 1
		ld-course-infobar : 1
		theme-post-content : 1
	sfwd-courses : 2
		jet-slider : 1
		ld-course-certificate : 1
		ld-course-content : 1
		ld-course-infobar : 1
		template : 1
		theme-post-content : 1
	jet-woo-builder-cart : 0
		jet-cart-table : 1
	jet-listing-items : 0
		button : 3
		divider : 1
		heading : 11
		icon-list : 1
		image : 3
		jet-listing-dynamic-field : 4
		jet-listing-dynamic-link : 2
		jet-listing-dynamic-terms : 5
	wp-page : 34
		button : 36
		divider : 40
		form : 8
		heading : 238
		icon : 49
		icon-box : 6
		icon-list : 17
		image : 63
		image-carousel : 2
		jet-ajax-search : 4
		jet-animated-box : 48
		jet-blog-smart-tiles : 5
		jet-engine-booking-form : 4
		jet-listing-grid : 5
		jet-slider : 1
		jet-smart-filters-checkboxes : 7
		jet-smart-filters-date-period : 2
		jet-smart-filters-remove-filters : 3
		jet-smart-filters-search : 3
		jet-smart-filters-select : 3
		jet-tabs : 6
		jet-unfold : 4
		shortcode : 13
		slides : 1
		spacer : 57
		template : 5
		text-editor : 70
	wp-post : 2
		text-editor : 3
	search-results : 1
		heading : 5
		image : 1
		jet-ajax-search : 1
		jet-blog-smart-tiles : 1
		jet-listing-grid : 1
		jet-smart-filters-checkboxes : 1
		jet-smart-filters-remove-filters : 1
		jet-smart-filters-search : 1
		jet-smart-filters-select : 1


== Log ==
	
PHP: showing 19 of 19PHP: 2021-03-09 21:06:57 [notice X 6][C:\Bitnami\wordpress\apps\wordpress\htdocs\wp-content\plugins\elementor\includes\managers\image.php::86] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-03-09 21:20:41 [notice X 108][C:\Bitnami\wordpress\apps\wordpress\htdocs\wp-content\plugins\elementor\core\common\modules\ajax\module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-03-10 23:05:22 [error X 29][/var/www/wp-content/plugins/elementor/includes/utils.php::580] Uncaught Error: Call to undefined function Elementor\mb_strtolower() in /var/www/wp-content/plugins/elementor/includes/utils.php:580
Stack trace:
#0 /var/www/wp-content/plugins/elementor-pro/modules/custom-attributes/module.php(110): Elementor\Utils::parse_custom_attributes()
#1 /var/www/wp-includes/class-wp-hook.php(292): ElementorPro\Modules\CustomAttributes\Module->render_attributes()
#2 /var/www/wp-includes/class-wp-hook.php(316): WP_Hook->apply_filters()
#3 /var/www/wp-includes/plugin.php(484): WP_Hook->do_action()
#4 /var/www/wp-content/plugins/elementor/includes/base/element-base.php(863): do_action()
#5 /var/www/wp-content/plugins/elementor/includes/base/widget-base.php(423): Elementor\Element_Base->add_render_attributes()
#6 /var/www/wp-content/plugins/elementor/includes/base/element-base.php(675): Elementor\Widget_Base->add_render_attributes()
#7 /var/www/wp-content/plugins/elementor/includes/base/element-base.php(911): Elementor\Element_Base->print_element()
#8 /var/www/wp-content/plugins/elementor [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-03-12 20:17:13 [error X 1][/var/www/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::281] Uncaught Error: Call to a member function update_meta() on bool in /var/www/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:281
Stack trace:
#0 /var/www/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(162): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions()
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions()
#2 /var/www/wp-content/plugins/elementor/core/common/modules/ajax/module.php(171): call_user_func()
#3 /var/www/wp-includes/class-wp-hook.php(292): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request()
#4 /var/www/wp-includes/class-wp-hook.php(316): WP_Hook->apply_filters()
#5 /var/www/wp-includes/plugin.php(484): WP_Hook->do_action()
#6 /var/www/wp-admin/admin-ajax.php(187): do_action()
#7 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-03-15 16:22:22 [notice X 13][/var/www/wp-content/plugins/elementor/includes/managers/image.php::80] Undefined offset: 2 [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-03-16 00:52:23 [notice X 68][/var/www/wp-content/plugins/elementor-pro/modules/query-control/module.php::382] Undefined index: display [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-03-17 12:57:27 [warning X 203][/var/www/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php::101] file_get_contents(/var/www/wp-content/uploads/2021/03/bullet.svg): failed to open stream: No such file or directory [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-03-17 15:05:43 [notice X 32][/var/www/wp-content/plugins/elementor-pro/modules/forms/classes/form-base.php::212] Undefined index: allow_multiple [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-03-18 14:27:27 [notice X 145][/var/www/wp-content/plugins/elementor-pro/modules/forms/submissions/actions/save-to-database.php::82] Undefined index: field_type [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-03-19 19:00:09 [notice X 697][/var/www/wp-content/plugins/elementor/includes/base/controls-stack.php::1405] Trying to access array offset on value of type null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-03-20 02:09:46 [error X 2][/var/www/wp-content/plugins/elementor/core/common/modules/ajax/module.php::265] Allowed memory size of 134217728 bytes exhausted (tried to allocate 1175552 bytes) [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-03-20 02:10:03 [error X 1][/var/www/wp-content/plugins/elementor/core/common/modules/ajax/module.php::265] Allowed memory size of 134217728 bytes exhausted (tried to allocate 17691560 bytes) [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-03-22 18:53:51 [notice X 16][/var/www/wp-content/plugins/elementor-pro/modules/query-control/module.php::738] Trying to get property 'name' of non-object [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-04-01 16:51:33 [notice X 5][/var/www/wp-content/plugins/elementor/modules/system-info/reporters/user.php::69] Undefined offset: 0 [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-04-05 18:57:02 [notice X 11][/var/www/wp-content/plugins/elementor-pro/modules/forms/classes/ajax-handler.php::244] Undefined index: post_id [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-04-06 20:12:38 [notice X 51][/var/www/wp-content/plugins/elementor-pro/modules/forms/classes/form-base.php::162] Undefined index: required [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-04-15 22:37:32 [warning X 1][/home/customer/www/staging4.*confidential*.com/public_html/wp-content/plugins/elementor/modules/safe-mode/module.php::124] rmdir(/home/customer/www/staging4.*confidential*.com/public_html/wp-content/mu-plugins): Directory not empty [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-04-19 21:13:24 [notice X 2][/home/customer/www/staging4.*confidential*.com/public_html/wp-content/plugins/elementor-pro/modules/posts/skins/skin-content-base.php::207] Trying to get property 'ID' of non-object [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-04-23 12:38:51 [notice X 5][/home/customer/www/staging4.*confidential*.com/public_html/wp-content/plugins/elementor-pro/modules/forms/classes/form-record.php::230] Undefined index: field_label [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 9 of 92021-03-10 13:05:38 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.1.3',
  'to' => '3.1.4',
)]
2021-03-17 12:55:18 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.1.1',
  'to' => '3.2.0',
)]
2021-03-23 14:24:34 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.2.0',
  'to' => '3.2.1',
)]
2021-04-22 11:34:49 [info] elementor::elementor_updater Started 
2021-04-22 11:34:49 [info] Elementor/Upgrades - _v_3_2_0_migrate_breakpoints_to_new_system Start  
2021-04-22 11:34:49 [info] Elementor/Upgrades - _v_3_2_0_migrate_breakpoints_to_new_system Finished 
2021-04-22 11:34:49 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.1.4',
  'to' => '3.2.1',
)]
2021-04-22 11:34:49 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.1.4',
  'to' => '3.2.1',
)]
2021-04-27 11:57:09 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.2.1',
  'to' => '3.2.2',
)]


				
```
</details>

**Comment by maurocarreraok** [comment]
@shilo-ey
